playerMat.scss

```diff
@@ -30,7 +30,7 @@
 }
 
 #wall .tile[occupied=false]:not([type="-1"]):not(.option) {
-	opacity: .1;
+	opacity: .2;
```

Comment:
These unoccupied wall tiles looked really really dark and indistinguishable to me when using Chrome's vision deficiency emulation. Even with the patterned backgrounds.